src/seedu/addressbook/storage/StorageFile.java

```diff
@@ -152,5 +154,11 @@ public AddressBook load() throws StorageOperationException {
     public String getPath() {
         return path.toString();
     }
-
+    
+    public void checkStorageFileExists() throws StorageOperationException{
```

Comment:
Throwing an exception from this method seems like an overkill. A `boolean isStorageFilePresent()` is more suitable?